files/image_config/interfaces/interfaces-config.sh

```diff
@@ -60,8 +60,7 @@ for intf_pid in $(ls -1 /var/run/dhclient*.Ethernet*.pid 2> /dev/null); do
     [[ -f ${intf_pid} ]] && kill `cat ${intf_pid}` && rm -f ${intf_pid}
 done
 
-[[ -f /var/run/resolvconf/interface/eth0.dhclient ]] && rm -f /var/run/resolvconf/interface/eth0.dhclient
-[[ -f /var/run/resolvconf/interface/eth0.ip6.dhclient ]] && rm -f /var/run/resolvconf/interface/eth0.ip6.dhclient
+/usr/bin/resolv-config.sh cleanup
```

Comment:
If user restarts interfaces-config.sh alone, will it lead to static or dynamic dns getting removed? In ZTP we restart interfaces-config service alone in some scenarios. Will it result in any resolution failure?

Reply:
We run cleanup for dynamic configuration only.
If we have static DNS entries configured they won't be affected